plugin/completion/compiler_variant.py

```diff
@@ -72,7 +72,7 @@ class ClangClCompilerVariant(ClangCompilerVariant):
         error_regex (re): regex to find contents of an error
     """
     init_flags = ["-c", "-fsyntax-only"]
-    include_prefixes = ["-I ", "/I ", "-msvc ", "/imsvc "]
+    include_prefixes = ["-I", "-I ", "/I ", "-msvc ", "/imsvc "]
```

Comment:
There shouldn't be spaces in imsvc prefixes or /I because those are supported both with and without spaces.

Also, what's the point of including `-I ` if the `-I ` will catch it first anyway?

(Style nit: Sometimes single and sometimes double quotes are used.)